dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -753,7 +753,9 @@ public void handleInfo(String entryPath) {
                 errorMessage("This container is not published.", Client.COMMAND_ERROR);
             } else {
 
-                Date lastBuild = container.getLastBuild();
+                long lastBuildLong = container.getLastBuild();
+                Date lastBuild = new Date(lastBuildLong);
+
                 Date dateUploaded = null;
                 if (lastBuild != null) {
```

Comment:
I thought I saw a warning about this from Codacy -- lastBuild can't be null.

Reply:
I did not see this in Codacy; why couldn't lastBuild in fact be null? Comment at dockstore-webservice/src/main/resources/swagger.yaml:6666 says it can?

Reply:
I think the idea is that even if `getLastBuild()` comes back from our API as null (not that Codacy understands our swagger probably) 
`Date lastBuild` won't be null because `new Date(null)` won't result in null (probably either exception or an epoch date of 1970ish)

Reply:
1. Sorry it was [lgtm](https://lgtm.com/projects/g/dockstore/dockstore-cli/rev/pr-1efc905f899a6835780c6ce185897bc84016c7b3), not Codacy -- I had had a vague recollection of having seen some email mentioning it.
2. `container.getLastBuild()` returns a Long, not a long, so on line 756 you should make it a Long. And because it's Long, you should do a null check there.

Reply:
fixed